app/models/memory.rb

```diff
@@ -8,4 +13,30 @@ class Memory < ActiveRecord::Base
   validates :name, presence: true
   validates :keywords, presence: true
   validates :description, presence: true
+
+  def related_memories_sorted
+    related_memories.group("memories.id").count.sort_by { |k,v| v }.reverse.map { |k,v| Memory.find(k) }
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.<br>Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used.<br>Space missing after comma.